markets/spot-market/contracts/modules/AtomicOrderModule.sol

```diff
@@ -99,6 +112,10 @@ contract AtomicOrderModule is IAtomicOrderModule {
             ITokenModule(spotMarketFactory.usdToken).transfer(msg.sender, returnAmount);
         }
 
+        if (returnAmount < minAmountReceived) {
+            revert InsufficientReturnAmount(minAmountReceived, returnAmount);
```

Comment:
Rename to InsufficientAmountReceived ?

Reply:
Also, maybe we move this up under the process fee call? Would save a little gas I think?